tests/test_manifest.py

```diff
@@ -94,7 +94,7 @@ def test_manifest_creation():
     component_id = "component_id"
 
     manifest = Manifest.create(base_path=base_path, run_id=run_id, component_id=component_id)
-    manifest.add_subset("images", [("width", Type.int32), ("height", Type.int32)])
+    manifest.add_subset("images", [("width", "int32"), ("height", "int32")])
```

Comment:
what' the reason for changing this? Shouldn't it be a datatype

Reply:
The reason for this was that, when getting the fields of an output subset of the component spec, this just gives me a string:

```
>>> from express.component_spec import ComponentSpec
>>> spec = ComponentSpec("fondant_component.yaml")
>>> spec.output_subsets
mappingproxy({'images': ComponentSubset({'fields': {'data': {'type': 'binary'}, 'width': {'type': 'int16'}, 'height': {'type': 'int16'}}}, 'captions': ComponentSubset({'fields': {'data': {'type': 'utf8'}}}})
>>> spec.output_subsets["images"]
ComponentSubset({'fields': {'data': {'type': 'binary'}, 'width': {'type': 'int16'}, 'height': {'type': 'int16'}}}
>>> spec.output_subsets["images"].fields["data"]
Field(name='data', type='binary')
>>> spec.output_subsets["images"].fields["data"].type
'binary'
```
It doesn't give me an object of type `Type`